ENGAGEHF/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+//  DashboardView.swift
+//  ENGAGEHF
+//
+//  Created by Nick Riedman on 4/3/24.
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct Greeting: View {
+    @State private var dateString: String?
+    
+    var body: some View {
+        HStack {
+            Text("Hello, world!")
+                .font(.title2)
+                .accessibilityLabel(Text("DASHBOARD_GREETING"))
```

Comment:
As noted in some other review comments: You don't need explicit accessibility labels for every element in a view, SwiftUI generates a lot of them for you automatically and in a good quality.